src/smc-webapp/editor.coffee

```diff
@@ -1147,8 +1147,11 @@ class exports.Editor
             # responsive mode
             @project_page.destroy_sortable_file_list()
             width = "50%"
+            $("#file_info_button").addClass("btn-lg")
```

Comment:
1. The selector `$("#file_info_button")` changes file_info_button EVERYWHERE, not just in this particular document.  You might want `@element.find("#file_info_button")`, except that...

2. The whole idea of CSS responsive mode is that the size of the button is determined by CSS entirely rather than javascript.  What you're doing here is just directly fighting against how CSS is meant to work.   Please see http://getbootstrap.com/css/#responsive-utilities